public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -304,6 +304,9 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> impleme
         slug: result.slug,
         folderUid: folderUid,
         version: result.version,
+        // this is to just keep this k8s prop in sync with serializer
+        // this prop should be removed from scene and live in serializer's metadata
+        k8s: this.serializer.getK8SMetadata(),
```

Comment:
I guess this metadata is in a state to be able to subscribe to rerender components when the metadata changes 🤔 
I we don't have a way to subscribe the serializer metadata from there, we need this k8s here. So I find the comment confusing because we don't know if we ever move this into the serializer.

Reply:
🤔 do we still need to passs this k8s in the DashboardScene? I saw you already fixed the error by removing the "resourceVersion" in v1.ts